ci-operator/step-registry/baremetalds/sno/rename/baremetalds-sno-rename-commands.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+cat >"${SHARED_DIR}"/recert-rename-sno.sh << "EOF"
+#!/usr/bin/env bash
+
+export PREVIOUS_CLUSTER_NAME="${PREVIOUS_CLUSTER_NAME:-test-infra-cluster}"
+export PREVIOUS_BASE_DOMAIN="${PREVIOUS_BASE_DOMAIN:-redhat.com}"
+export CLUSTER_NAME="${CLUSTER_NAME:-another-name}"
+export BASE_DOMAIN="${BASE_DOMAIN:-another.domain}"
+export SINGLE_NODE_IP="${SINGLE_NODE_IP:-192.168.127.10}"
+
+# assisted-test-infra sets up 2 network interfaces that compete with each other
+# when setting the NODE_IP and KUBELET_NODEIP. Use KUBELET_NODEIP_HINT to
+# ensure the correct interface is chosen.
+#
+# https://access.redhat.com/articles/6956852
+ssh -o UserKnownHostsFile=/dev/null \
+  -o StrictHostKeyChecking=no \
+  -o LogLevel=ERROR \
+  "core@${SINGLE_NODE_IP}" \
+  "echo 'KUBELET_NODEIP_HINT=192.168.127.1' | sudo tee /etc/default/nodeip-configuration"
+
+VMS=( $( virsh list --all --name ) )
+for vm in ${VMS[@]}; do
+  virsh reboot ${vm}
```

Comment:
What are we we rebooting VMs for?

Reply:
We don't need this reboot, if the kube API was down (due to etcd being down, trying to use the wrong NODE_IP) the previous step (i.e. the SNO setup) would have failed. I'll remove the reboot :+1: